client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -20,127 +20,274 @@
 import java.util.Set;
 
 import com.google.common.collect.Sets;
-
-import org.apache.uniffle.client.util.RssClientConfig;
+import org.apache.spark.internal.config.ConfigBuilder;
+import org.apache.spark.internal.config.ConfigEntry;
 
 public class RssSparkConfig {
 
+  public static final class SparkConfigBuilder {
+    public static ConfigBuilder configBuilder;
+
+    public static ConfigBuilder key(String key) {
+      configBuilder = new ConfigBuilder(key);
+      return configBuilder;
+    }
+  }
+
+  public static class RssConfigEntry<T> {
+    public ConfigEntry<Object> entry;
+    public String key;
+
+    public RssConfigEntry(ConfigEntry<Object> entry) {
+      this.entry = entry;
+      this.key = entry.key();
+    }
+
+    public T getValue() {
+      return (T) entry.defaultValue().get();
+    }
+  }
+
+  public static final RssConfigEntry<Integer> RSS_PARTITION_NUM_PER_RANGE = new RssConfigEntry(
+      SparkConfigBuilder
+          .key("spark.rss.partitionNum.per.range")
+          .intConf()
+          .createWithDefault(1));
```

Comment:
Could we modify this?
From
```
createWithDefault(1))
```
to
```
createWithDefault(RssClientConfig.RSS_PARTITION_NUM_PER_RANGE_DEFAULT_VALUE);
```
Because it's more easy to modify default value in the future if we need.

Reply:
Sure, is that the only parameter you need to change?

Reply:
All shared parameters are set this way.

Reply:
Done.